appengine/go11x/warmup/app.yaml

```diff
@@ -1,9 +1,8 @@
-runtime: go111
+runtime: go114
 
 inbound_services:
   - warmup
 
 handlers:
   - url: /_ah/warmup
     script: auto
-    login: admin
```

Comment:
Given auto is the default behavior, is the handlers block needed in Gen2?

Reply:
It seems to be ignored in Gen2. But some people may still look at this sample for Gen1, so I left it here.